Samples/Server/Unity-FractalStream/Assets/Scripts/WebRTCEyes.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.Toolkit.ThreeD
+{
+    /// <summary>
+    /// Represents 1 or 2 eyes used as the viewport for a <see cref="WebRTCServer"/>
+    /// </summary>
+    public class WebRTCEyes : MonoBehaviour
+    {
+        /// <summary>
+        /// Represents the possible number of eyes
+        /// </summary>
+        public enum EyeCount
+        {
+            One,
+            Two
+        }
+
+        /// <summary>
+        /// Total number of eyes
+        /// </summary>
+        public EyeCount TotalEyes = EyeCount.One;
+
+        /// <summary>
+        /// How many eyes we currently render with
+        /// </summary>
+        public EyeCount VisibleEyes = EyeCount.One;
+
+        /// <summary>
+        /// Camera for first (left-most) eye
+        /// </summary>
+        public Camera EyeOne;
+
+        /// <summary>
+        /// Camera for second (from the left) eye
+        /// </summary>
+        /// <remarks>
+        /// In a 2 eye (most common) scenario this is also known as the right eye
+        /// </remarks>
+        public Camera EyeTwo;
+
+        /// <summary>
+        /// The eye count from the last <see cref="SetupActiveEyes"/> call
+        /// </summary>
+        private EyeCount? lastSetupVisibleEyes = null;
+
+        /// <summary>
+        /// List of all camera for all eyes
+        /// </summary>
+        public IList<Camera> AllEyeCameras

```

Comment:
nit: is this really needed?

Reply:
@bengreenier I agree, we don't need this class at all. The stereo distance should be set from the Unity editor and the left/right eye will always be present in the scene. We can easily get the reference to them without the need of a custom class but we need to make sure we switch between a mono/stereo streaming.